src/copy_from_template.py

```diff
@@ -134,6 +137,25 @@ def get_folder(created_entities, folder_name):
     return folder["entity"]
 
 
+def get_wiki_sub_page(syn, wiki_project, wiki_title):
```

Comment:
This is fine, but a more robust solution would be to parameterize this function by the wiki ID, rather than the wiki title.

Reply:
What wiki ID are you referring to? The 620176 in https://www.synapse.org/#!Synapse:syn26546076/wiki/620176 for example?

Reply:
Yes

Reply:
I see. The goal of this function is to get the wiki subpage object that contains the wiki subpage id to be used in the copyWiki functions in the script.

And because getWikiHeaders() used in this function would only give subpage objects with the following fields: id, title and parentId, I had to use wiki title as a filter so we can pull the id. Otherwise, the alternative was to just add two new arguments to the script to have the user specify the specific parquet wiki id and main project wiki id directly.